packages/relay/src/TransactionManager.ts

```diff
@@ -213,16 +257,17 @@ data         | 0x${transaction.data.toString('hex')}
     }
   }
 
-  _resolveNewGasPrice (oldGasPrice: number): { newGasPrice: number, isMaxGasPriceReached: boolean } {
+  _resolveNewGasPrice (oldMaxFee: number, oldMaxPriorityFee: number): { newMaxFee: number, newMaxPriorityFee: number, isMaxGasPriceReached: boolean } {
     let isMaxGasPriceReached = false
-    let newGasPrice = oldGasPrice * this.config.retryGasPriceFactor
+    let newMaxFee = oldMaxFee * this.config.retryGasPriceFactor
```

Comment:
max fee should not factor - it should stay using the same configured max, and will cause to wait until the fee goes below it.
only priority should be factored.